src/components/Filters.js

```diff
@@ -1,6 +1,36 @@
 import React from "react";
+import { connect } from "react-redux";
+import {
+  searchByTitle,
+  searchByCity,
+  searchByCategory,
+  sortByNewest,
+  sortByClosest,
+} from "../actions/filters";
 
 class Filters extends React.Component {
+  onTitleFilterChange = e => {
+    this.props.searchByTitle(e.target.value);
+  };
+
+  onCityFilterChange = e => {
+    this.props.searchByCity(e.target.value);
+  };
+
+  onCategoryFilterChange = e => {
+    this.props.searchByCategory(e.target.value);
+  };
+
+  onSortByChange = e => {
+    const sortBy = e.target.value;
+    if (sortBy === "newest") {
+      this.props.sortByNewest(sortBy);
+    }
+    if (sortBy === "closest") {
```

Comment:
I would be pleased if you would separate cons and both `if` statements with an empty space :)